com.unity.UnityHub.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.unity.UnityHub.desktop</id>
```

Comment:
Don’t use a `.desktop` suffix, instead use a proper ID and add a `<launchable>` entry linking to the desktop file: https://github.com/flathub/flathub/wiki/AppData-Guidelines#id